src/ray/id.cc

```diff
@@ -81,6 +81,12 @@ bool UniqueID::operator==(const UniqueID &rhs) const {
   return std::memcmp(data(), rhs.data(), kUniqueIDSize) == 0;
 }
 
+size_t UniqueID::hash() const {
+  size_t result;
+  std::memcpy(&result, id_, sizeof(size_t));
```

Comment:
@stephanie-wang this hashing scheme will likely be a problem with the new way xray is encoding information in the Object IDs, right?

I don't want to address it in this PR, since this PR should not change the hash function, but just want to bring it up.

Reply:
I reused the old hash function in the PR. We can change the hash function when we change the Object IDs' encoding. 
And we can provide different hash function for different ID. We also changed the hash function for the plasma ObjectID in our internal branch, it will not affect it.

Reply:
Do you have any idea how to repro the https://travis-ci.org/ray-project/ray/jobs/371987419, I have run the "python test/runtest.py" already, but no error raised.

Reply:
You can either copy the diff from travis or run some variant of this:

```
~/ray/.travis/git-clang-format --binary clang-format --commit 3c76461b225eb91a86cc0802e20a04e65b5b408a --diff --exclude '(.*thirdparty/|.*redismodule.h|.*webui*)'
```

where 3c76461b225eb91a86cc0802e20a04e65b5b408a should be the parent commit of your PR.